src/SFA.DAS.Commitments.Support.SubSite/assets/fontawesome-5.0.2/LICENSE.txt

```diff
@@ -0,0 +1,34 @@
+Font Awesome Free License
```

Comment:
Are these assets (css/js etc.) the same ones that the support console wrapper uses?
I understand that when wrapped by the support console, this component will be returning html fragments (and will presumably inherit the assets of the wrapper), so for development and testing we should be using the same assets of the wrapper.